packages/@jsii/benchmarks/bin/benchmark.ts

```diff
@@ -0,0 +1,86 @@
+import * as fs from 'fs-extra';
+import * as yargs from 'yargs';
+
+import { benchmarks } from '../lib';
+import { Benchmark } from '../lib/benchmark';
+
+/**
+ * Format of benchmark output used by continous benchmarking action.
+ * See [documentation](https://github.com/benchmark-action/github-action-benchmark/blob/master/README.md) for details
+ */
+interface ResultsJson {
+  /**
+   * The name of the benchmark
+   */
+  name: string;
+
+  /**
+   * The unit of measure, usually seconds
+   */
+  unit: string;
+
+  /**
+   * The result of the measurement, usually an average over x iterations
+   */
+  value: number;
+
+  /**
+   * The variance of all runs
+   */
+  range: number;
+
+  /**
+   * Extra information about the benchmark, displayed in a tooltip
+   */
+  extra: string;
+}
+
+(async () => {
+  const argv = yargs
```

Comment:
Add `await` here (the next version of `yargs` types `argv` as `Args | Promise<Args>` 😒)

```suggestion
  const argv = await yargs
```